src/main/java/io/jenkins/pluginhealth/scoring/service/PluginService.java

```diff
@@ -37,6 +39,24 @@ public PluginService(PluginRepository pluginRepository) {
         this.pluginRepository = pluginRepository;
     }
 
+    public void updateDatabase(Plugin plugin) {
+        Optional<Plugin> pluginFromDatabase = pluginRepository.findByName(plugin.getName());
+
+        if (pluginFromDatabase.isPresent()) {
+            if (pluginFromDatabase.get().getReleaseTimestamp() != plugin.getReleaseTimestamp()) {
```

Comment:
This condition is always true because `releaseTimestamp` of the plugin fetched from the DB has the zone ID of `Asia/Kolkata` and the plugin fetched from update-center.json file has zone ID of `UTC`.

I tried to include these changes one-by-one in the application.properties file because I want JPA to store timestamps with `UTC` zoneID:

* appended jdbc URL with: `?useLegacyDatetimeCode=false&serverTimezone=UTC`
* spring.jpa.properties.hibernate.jdbc.time_zone=UTC
* spring.jackson.time-zone=UTC

But I still see that the zone ID of the fetched plugins is not `UTC`. But the `releaseTimestamp` shows in UTC format when I run a select-query on the DB.